### ISSUE TITLE
Research tabs accessibility

### ISSUE THREAD

**Issue by nakayan5** [yamada-ui/yamada-ui#1509]
### Description

read [this article](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/) and research tabs accessibility

modify component if necessary

### Problem Statement/Justification

none

### Alternatives

- [ ] #1636

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

**Comment by nakayan5** [comment]
@yamada-ui/tabs is applicable to this issue

**Comment by nakayan5** [comment]
@illionillion 
please double check

**Comment by illionillion** [comment]
@nakayan5 
> The active tab element has the state [aria-selected](https://w3c.github.io/aria/#aria-selected) set to true and all other tab elements have it set to false.
aria-selected is set to true but not false and aria-selected itself is not shown, is this still a problem?

**Comment by nakayan5** [comment]
@hirotomoyamada 
The above checklist is the status of correspondence.
Which ones do you  support?

**Comment by hirotomoyamada** [comment]
@nakayan5 

> The active tab element has the state [aria-selected](https://w3c.github.io/aria/#aria-selected) set to true and all other tab elements have it set to false.

`aria-selected` should be `false`.

**Comment by koralle** [comment]
@nakayan5 @illionillion 

For your reference, the HTML structure of the [React Aria `Tab` component](https://react-spectrum.adobe.com/react-aria/Tabs.html) is as follows:

```html
<div
  id="react-aria2733588558-:r14:"
  aria-label="History of Ancient Rome"
  role="tablist"
  aria-orientation="horizontal"
  class="react-aria-TabList"
  data-rac=""
  data-orientation="horizontal"
>
  <div
    tabindex="0"
    data-key="FoR"
    id="react-aria2733588558-:r14:-tab-FoR"
    aria-selected="true"
    role="tab"
    class="react-aria-Tab"
    data-rac=""
    aria-controls="react-aria2733588558-:r14:-tabpanel-FoR"
    data-selected="true"
  >
    Founding of Rome
  </div>
  <div
    tabindex="-1"
    data-key="MaR"
    id="react-aria2733588558-:r14:-tab-MaR"
    aria-selected="false"
    role="tab"
    class="react-aria-Tab"
    data-rac=""
  >
    Monarchy and Republic
  </div>
  <div
    tabindex="-1"
    data-key="Emp"
    id="react-aria2733588558-:r14:-tab-Emp"
    aria-selected="false"
    role="tab"
    class="react-aria-Tab"
    data-rac=""
  >
    Empire
  </div>
</div>
```

![image](https://github.com/yamada-ui/yamada-ui/assets/33865215/974bc584-954e-42f8-b88f-4b3dbc0b110a)

**Comment by nakayan5** [comment]
created issue
https://github.com/yamada-ui/yamada-ui/issues/1636
